addon/templates/components/freestyle-dynamic.hbs

```diff
@@ -0,0 +1,24 @@
+{{#freestyle-annotation}}
+  <h3 class="FreestyleDynamic-header">Dynamic Properties:</h3>
```

Comment:
Would be cool if this is not hardcoded, so we can use title for certain component.